banzai/qc/header_checker.py

```diff
@@ -130,14 +144,17 @@ def check_ra_range(self, image):
 
         """
         ra_value = image.header['CRVAL1']
-
         logging_tags = logs.image_config_to_tags(image, self.group_by_keywords)
-
-        if isinstance(ra_value, float):
-
-            if (ra_value > 360) | (ra_value < 0):
+        try:
+            bad_ra_value = (ra_value > self.RA_MAX) | (ra_value < self.RA_MIN)
+            if bad_ra_value:
                 sentence = 'The header CRVAL1 key got the unexpected value : ' + str(ra_value)
                 self.logger.error(sentence, extra=logging_tags)
+            self.save_qc_results({"HeaderBadRAValue": bad_ra_value}, image)
```

Comment:
bad_ra_value should be renamed to something indicative of a flag, like is_bad_ra_value